app/code/Magento/LoginAsCustomerGraphQl/etc/schema.graphqls

```diff
@@ -0,0 +1,30 @@
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
+
+type Mutation {
+    generateCustomerTokenAsAdmin(
+        input: GenerateCustomerTokenAsAdminInput!
+    ): GenerateCustomerTokenAsAdminOutput
+    @resolver(class: "Magento\\LoginAsCustomerGraphQl\\Model\\Resolver\\RequestCustomerToken")
+    @doc(description: "Request a customer token so that an administrator can perform remote shopping assistance")
+}
+
+input GenerateCustomerTokenAsAdminInput {
+    customer_email: String! @doc(description: "The email address of the customer requesting remote shopping assistance")
+}
+
+type GenerateCustomerTokenAsAdminOutput {
+    customer_token: String! @doc(description: "The generated customer token")
+}
+
+type Customer {
+    allow_remote_shopping_assistance: Boolean! @doc(description: "Indicates whether the customer has enabled remote shopping assistance")
+}
+
+input CustomerCreateInput {
+    allow_remote_shopping_assistance: Boolean @doc(description: "Indicates whether the customer has enabled remote shopping assistance")
+}
+
+input CustomerUpdateInput {
+    allow_remote_shopping_assistance: Boolean @doc(description: "Indicates whether the customer has enabled remote shopping assistance")
+}
```

Comment:
Resolvers using these inputs will need to be updated to ensure allow_remote_shopping_assistance is persisted.

Reply:
Done

Reply:
I don't see the changes for this. The createCustomer query needs to be updated so that allow_remote_shopping_assistance is persisted

`mutation {
  createCustomerV2(input: { email: "test@example.com" firstname: "Test" lastname: "Magento" password: "T3stP4ssw0rd" allow_remote_shopping_assistance: true})
{
  customer { firstname lastname email allow_remote_shopping_assistance }
}
}`